be/src/exec/tablet_info.cpp

```diff
@@ -171,7 +171,6 @@ Status OlapTablePartitionParam::init() {
         }
         _partition_slot_descs.push_back(it->second);
     } else {
-        DCHECK(_t_param.__isset.partition_columns);
```

Comment:
Please add `else if` to make it more clear and definite：

```
} else if (_t_param.__isset.partition_columns) {
xxxx
}
```

Although without this `else if`, it still works

Reply:
OK, update